packages/web-components/@sfx/sayt/test/unit/sayt.test.ts

```diff
@@ -58,16 +91,113 @@ describe('Sayt Component', () => {
       expect(removeEventListener).to.be.calledWith('click', sayt.processClick);
       expect(removeEventListener).to.be.calledWith('keydown', sayt.processKeyEvent);
     });
+
+    it('should unregister listener for searchbox change if searchbox is not given', () => {
+      const removeEventListener = stub(window, 'removeEventListener');
+      sayt.searchbox = '';
+
+      sayt.disconnectedCallback();
+
+      expect(removeEventListener).to.be.calledWith('sfx::searchbox_change', sayt.processSfxSearchboxChange);
+    });
+
+    it('should not unregister listener for input change if specified searchbox element exists', () => {
```

Comment:
Your tests are going to change and some of these will probably go away. If you implement the suggested changes about `toggleSearchboxListener`, your test will simply be asserting that the function is called.